checker/src/org/checkerframework/checker/index/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -463,6 +463,20 @@ private void addAnnotationForDivide(
                 result = ((LessThanLengthOf) numerator).divide(divisor.intValue());
             }
             result = result.glb(plusTreeDivideByVal(divisor.intValue(), numeratorTree));
+
+            // If the numerator is an array length access of an array with length greater than 1, and the divisor is
```

Comment:
The comment was just mixed up. Thanks for pointing it out. I've fixed it and I'm going to merge this PR.